pkg/apis/keepalive/v1beta1/register.go

```diff
@@ -0,0 +1,74 @@
+package v1beta1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// VirtualIP is a keepalived CRD specificiation for virtual IP addresses
+type VirtualIP struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+
+	Spec   VirtualIPSpec `json:"spec"`
+	Status Status        `json:"status"`
+}
+
+// VirtualIPSpec defines the spec of the CRD
+type VirtualIPSpec struct {
+	// IP virtual IP to use
+	IP string `json:"ip"`
+	// Port of the virtual
+	Port int `json:"port"`
```

Comment:
What's the purpose of this port? I didn't think VRRP operated at layer 3 to need a port.

Reply:
Removed